R/IDateTime.R

```diff
@@ -90,9 +90,11 @@ round.IDate <- function (x, digits=c("weeks", "months", "quarters", "years"), ..
   (setattr(as.integer(unclass(e1) + unclass(e2)), "class", c("IDate", "Date")))  # () wrap to return visibly
 }
 
-`-.IDate` <- function (e1, e2) {
-  if (!inherits(e1, "IDate"))
+`-.IDate` = function (e1, e2) {
```

Comment:
this part of the overall diff is not just `<-` removal... made this slight change in the second commit (coverage).

I was a bit wary of dispatching to `-.Date` since the result could then be `Date`. But the `if ( isReallyReal(e2) )` branch below does the same.